tests/model_registry/utils.py

```diff
@@ -315,3 +316,34 @@ def create_model_registry_instance(
         enable_database_upgrade=enable_database_upgrade,
         wait_for_resource=wait_for_resource,
     )
+
+
+def make_mysql_config(
+    db_service: Service,
+    db_secret: Secret,
+    ssl_ca: str,
+    db_resource_name: str = DB_RESOURCES_NAME,
+    port: int = 3306,
+) -> dict[str, Any]:
+    """
+    Factory to create a MySQL config dictionary for Model Registry.
+
+    Args:
+        db_service: The MySQL Service resource.
+        db_secret : The MySQL Secret resource.
+        ssl_ca: Path to the CA bundle file.
+        db_resource_name: Name of the DB resource (default: DB_RESOURCES_NAME).
+        port: Port number for the MySQL connection (default: 3306).
+
+    Returns:
+        dict: MySQL configuration dictionary.
+    """
+    return {
+        "host": f"{db_service.name}.{db_service.namespace}.svc.cluster.local",
+        "database": db_secret.string_data["database-name"],
+        "passwordSecret": {"key": "database-password", "name": db_resource_name},
+        "port": port,
+        "skipDBCreation": False,
+        "username": db_secret.string_data["database-user"],
+        "ssl_ca": ssl_ca,
+    }
```

Comment:
I like this a lot. I had added a similar fixture in my pr (`model_registry_mysql_config` in model_registry/conftest.py) - could you make ssl_ca optional and reuse this in that conftest as well?